src/components/user-profile.html

```diff
@@ -0,0 +1,78 @@
+<link rel="import" href="~@banno/polymer/polymer-element.html">
+
+<dom-module id="user-profile">
+  <template>
+    <style>
+      table {
+        box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+        background: #eef1f4;
+        border-radius: 3px;
+        padding: 30px;
+        margin-top: 10px;
+        width: 400px;
+      }
+      button {
+        background-color: DarkSlateBlue;
+        color: #fcfcff;
+        border-radius: 3px;
+        text-align: center;
+        padding: 10px;
+        margin-top: 20px;
+      }
+      .expand {
+        display: table-row;
+      }
+      .collapse {
+        display: none;
+      }
+    </style>
+    <h3>User Profile Component</h3>
+    <table>
+      <tr>
+        <td>ID:</td>
+        <td>[[id]]</td>
+      </tr>
+      <tr>
+        <td>Name:</td>
+        <td>[[firstName]] [[lastName]]</td>
+      </tr>
+      <tr class="expand">
+        <td>Phone:</td>
+        <td>[[phoneNumber]]</td>
+      </tr>
+      <tr class="expand">
+        <td>Email:</td>
+        <td>[[email]]</td>
+      </tr>
+      <tr>
+        <td>
+          <button onclick="cardExpandCollapse(this)">Collapse</button>
```

Comment:
We're gonna need to change this a bit when calling from a function
```suggestion
          <button on-click="cardExpandCollapse">Collapse</button>
```